### DISCUSSION TITLE
Infolist : Count of relation

### DISCUSSION THREAD

**Discussion by Hegabovic** [filamentphp/filament#9646] [Help] [answered]
Is there a way to display the count of those users instead of getting their names in a bullet representation?

![Screenshot from 2023-11-14 13-21-35](https://github.com/filamentphp/filament/assets/56083879/e9b10ef5-08fa-4d47-a4a0-1c7aecf659d1)

**Comment by sinnbeck** [comment, marked as answer]
Do you mean this?
```php
TextEntry::make('author_count')
   ->formatStateUsing(fn ($record) => $record->authors->count()),

**Reply by Hegabovic** [reply]
yes but is giving me , Call to a member function count() on null

**Reply by Hegabovic** [reply]
yeah yeah , you are correct , it work perfectly fine

**Reply by sinnbeck** [reply]
Did you make sure to load the relationship? Also make a fallback
```
TextEntry::make('authors')
   ->formatStateUsing(fn ($record) => $record->authors?->count() ?? 0),

**Comment by wallacemaxters** [comment]
Another way to do this is like this:

```php
public static function infolist(Infolist $infolist): Infolist
{
    $infolist->getRecord()->loadCount('authors');

    return $infolist->schema([
        // ...
        TextEntry::make('authors_count')->label('Authors')
    ]);
}
```
